### DISCUSSION TITLE
Unable to download/enable plugin

### DISCUSSION THREAD

**Discussion by epstein816** [coddingtonbear/obsidian-local-rest-api#100] [Q&A]
Trying to download the plugin via the browse function in obsidian I receive the following error.
![image](https://github.com/coddingtonbear/obsidian-local-rest-api/assets/45086669/3a80da5c-156d-4a3c-9c7f-c8b7ce734a85)

Similarly I am unable to enable the plugin when I tried to download it manually from github.
![image](https://github.com/coddingtonbear/obsidian-local-rest-api/assets/45086669/92f81079-8bf6-452f-a9f9-48d3f8f1e1b2)

I'm brand new to obsidian so its likely that my problem is some kind of user error. However, I was able to download and enable other plugins. Any assistance would be greatly appreciated.

**Comment by owrede** [comment]
I think the developer @coddingtonbear has to check this. Seems to be result of a failed build of 2.0.5 after fixing an issue in 2.0.4.

**Comment by coddingtonbear** [comment]
Sorry you all!  I was on vacation last week and absolutely was not going to pay for satellite internet access.  I've re-published a new build, and things should work once again!
